### ISSUE TITLE
Download and Error

### ISSUE THREAD

**Issue by mYAVka** [Lemon4ik666/Lethal-Company-Anti-Kick-Mod#3]
How to download mod?, i've found antikick.dll I took it in plugins folder, and it doesnt work.

**Comment by mYAVka** [comment]
If it have dependencies, add them.

**Comment by Lemon4ik666** [comment]
> Как скачать мод?, нашел antikick.dll, взял в папке с плагинами, а он не работает.

I don't know why this works for me

**Comment by Lemon4ik666** [comment]
> Если у него есть зависимости, добавьте их.

No only mod

**Comment by Lemon4ik666** [comment]
![image](https://github.com/Lemon4ik666/Lethal-Company-Anti-Kick-Mod/assets/48747799/f949ecc5-91c0-473b-ae5c-3b25d84651ba)

**Comment by mYAVka** [comment]
Yeah, i too have lethal config, and this mod shows in configuration.

**Comment by mYAVka** [comment]
justasomeguy12, my discord, lets talk in there.
